azurelinuxagent/ga/cgroupconfigurator.py

```diff
@@ -351,6 +351,22 @@ def _reset_agent_cgroup_setup(self):
             except Exception as err:
                 logger.warn("Error while resetting the quotas: {0}".format(err))
 
+        def _is_enforcement_supported(self, controller):
```

Comment:
this does not check whether enforcement is supported, but rather whether we are enforcing limits using that controller, right? if yes, can we rename the function

Reply:
It does two things
1) If Agent supports enforcement (like feature support) 
2) If above is true, it will check controller is mounted/enabled to enforce the limits

Reply:
where is 1 checked?

Reply:
i see, that is my point. this is not checking for "supported" (e.g. CPU Quotas are supported on V2); it's checking whether we are using the feature or not. name is misleading

Reply:
renamed to` _can_use_controller`

Reply:
thanks